### ISSUE TITLE
 Error occurred when installing the kruise-rollout addon in vela1.8

### ISSUE THREAD

**Issue by flysqrlboy** [kubevela/kubevela#5887]
**Describe the bug**

When running the command '**vela addon enable kruise-rollout**',
 an error will occur: '**Error: failed to dispatch dependent addon resource. Rendering addon application failed due to an error when loading the app template with CUE files. The error message is 'output.spec.policies.2.properties.clusters: conflicting values null and [...string]' (there is a type mismatch between null and list)**.

**To Reproduce**

**Expected behavior**

It can successfully install the kruise-rollout plugin in vela1.8
**Screenshots**

**KubeVela Version**

CLI Version: 1.8.0
Core Version: 1.8.0
GitRevision: git-fcd721ff
GolangVersion: go1.19.8

**Cluster information**

k8s v1.25
**Additional context**

**Comment by barnettZQG** [comment]
Maybe same with kubevela/velaux#773

**Comment by wangyikewxgm** [comment]
This is a bug in version 1.8. As a temporary workaround, you can use the following commands:
```
vela addon enable fluxcd
vela addon enable kruise-rollout
```

**Comment by wangyikewxgm** [comment]
https://github.com/kubevela/kubevela/pull/5865 is fixing this issue

**Comment by flysqrlboy** [comment]
> 

But It seems like it is not working.
After successfully installing fluxcd (using the command '**vela addon enable fluxcd**'),
 an error occurs when running the command '**vela addon enable kruise-rollout**' with the following error message: 

**W0420 13:06:28.969978 7779 render.go:259] Application name kruise-rollout will be overwritten with addon-kruise-rollout. Consider removing metadata.name in template. Error: Internal error occurred: failed calling webhook "mutating.core.oam.dev.v1beta1.applications": failed to call webhook: Post '[https://vela-core-webhook.vela-system.svc:443/mutating-core-oam-dev-v1beta1-applications?timeout=10s](https://vela-core-webhook.vela-system.svc/mutating-core-oam-dev-v1beta1-applications?timeout=10s)': no endpoints available for service "vela-core-webhook"**

**Comment by Somefive** [comment]
> > [#5887 (comment)](https://github.com/kubevela/kubevela/issues/5887#issuecomment-1515634695)
> 
> But It seems like it is not working. After successfully installing fluxcd (using the command '**vela addon enable fluxcd**'), an error occurs when running the command '**vela addon enable kruise-rollout**' with the following error message:
> 
> **W0420 13:06:28.969978 7779 render.go:259] Application name kruise-rollout will be overwritten with addon-kruise-rollout. Consider removing metadata.name in template. Error: Internal error occurred: failed calling webhook "mutating.core.oam.dev.v1beta1.applications": failed to call webhook: Post '[https://vela-core-webhook.vela-system.svc:443/mutating-core-oam-dev-v1beta1-applications?timeout=10s](https://vela-core-webhook.vela-system.svc/mutating-core-oam-dev-v1beta1-applications?timeout=10s)': no endpoints available for service "vela-core-webhook"**

This problem is because your KubeVela controller is not working correctly. You can use `kubectl get pods -n vela-system` to see if there is anything broken.

**Comment by flysqrlboy** [comment]
> 

you are  right. @Somefive 
The status of the pod named "kubevela-vela-core" is CrashLoopBackOff.

```
$kubectl get pods -n vela-system
NAME                                       READY   STATUS             RESTARTS         AGE
kubevela-cluster-gateway-b49d68b57-7n66r   1/1     Running            0                24h
apiserver-64464f66fd-4s2vs                 1/1     Running            0                24h
velaux-7c97d97c76-mxgk2                    1/1     Running            0                24h
kube-trigger-795946b87-6gdtl               1/1     Running            0                23h
kubevela-vela-core-599bccc969-6p69g        0/1     CrashLoopBackOff   82 (3m25s ago)   24h
```

the logs of the pod ( **leader election lost**):
```

I0420 11:38:56.091674       1 application_controller.go:469] "Application manifests has applied by workflow successfully" application="vela-system/addon-fluxcd" controller="application" resource_version="2821460" generation=1 publish_version="addon-20230420115118884" spanID="i-xbx38dyj.do-workflow-finish"
I0420 11:38:56.091754       1 application_controller.go:470] "[Finished]: i-xbx38dyj.do-workflow-finish(do-workflow-finish)" application="vela-system/addon-fluxcd" controller="application" resource_version="2821460" generation=1 publish_version="addon-20230420115118884" duration="6.879241677s" spanID="i-xbx38dyj.do-workflow-finish"
E0420 11:39:00.306419       1 leaderelection.go:330] error retrieving resource lock vela-system/kubevela: Get "https://10.43.0.1:443/apis/coordination.k8s.io/v1/namespaces/vela-system/leases/kubevela": context deadline exceeded
I0420 11:39:00.306654       1 leaderelection.go:283] failed to renew lease vela-system/kubevela: timed out waiting for the condition
E0420 11:39:00.306830       1 server.go:215] "Failed to run manager" err="leader election lost"
Error: leader election lost
```

What should I do？

**Comment by Somefive** [comment]
Great, I'm not sure why leader election lost happened so frequently, but it does have several ways for diagnosing it.

**Comment by wangyikewxgm** [comment]
This bug is being fixed by [pr](#5865) currently, please reopen this issue after the bug is fixed.
